tests/decrosstalk/data/ophys_plane_instantiation_data.json

```diff
@@ -0,0 +1,2766 @@
+{
```

Comment:
I think the naming convention for test data is to place it in a folder called `resources` instead of `data`